plugins/woocommerce/templates/emails/email-order-items.php

```diff
@@ -68,7 +68,7 @@
 							 * @param WC_Order_Item_Product $item      The item being displayed.
 							 * @since 2.1.0
 							 */
-							echo wp_kses_post( apply_filters( 'woocommerce_order_item_name', $item->get_name(), $item, false ) );
+							echo wp_kses_post( apply_filters( 'woocommerce_order_item_name', "<h3 style='font-size: inherit;'>" . $item->get_name() . '</h3>', $item, false ) );
```

Comment:
Why this change? The `'woocommerce_order_item_name'` filter should receive only the name, without any HTML markup (it's used around the codebase and needs to be consistent in what argument is passed), but I don't see the reason to add the `<h3>` in the first place.